public/script.js

```diff
@@ -73,11 +71,11 @@ function collap(room_id) {
   room_id = convertIntoId(room_id);
   var height = $(`#${room_id}-msg`).find('.Participants').css('opacity');
   if(height == '0') {
-   $(`#${room_id}-msg`).find('.Participants').css({"opacity":"1" , "z-index":"10"});
-  $(`#${room_id}-msg`).find('.btn').addClass('viewUsers');
+    $(`#${room_id}-msg`).find('.Participants').css({"opacity":"1" , "z-index":"10"});
+    $(`#${room_id}-msg`).find('.btn').addClass('viewUsers');
   } else {
-   $(`#${room_id}-msg`).find('.Participants').css({"opacity":"0" , "z-index":"-10"});
-  $(`#${room_id}-msg`).find('.btn').removeClass('viewUsers');
+    $(`#${room_id}-msg`).find('.Participants').css({"opacity":"0" , "z-index":"-10"});
+    $(`#${room_id}-msg`).find('.btn').removeClass('viewUsers');
```

Comment:
You should change the variable `height` to `opacity`, and remove unnecessary styling changes.

Reply:
I never touched this code.

Reply:
I understand that. Leave it for now.